pycheribuild/projects/cross/qt5.py

```diff
@@ -753,6 +755,10 @@ class BuildICU4C(CrossCompileAutotoolsProject):
     native_install_dir = DefaultInstallDir.CHERI_SDK
     make_kind = MakeCommandKind.GnuMake
     needs_native_build_for_crosscompile = True
+    supported_architectures = list(
+        set(CompilationTargets.ALL_SUPPORTED_CHERIBSD_AND_HOST_TARGETS) |
```

Comment:
Are you sure you intend to build against a hybrid rootfs and not the lib64 libraries in a purecap rootfs? In the latter case I'd suggest `    supported_architectures = CompilationTargets.ALL_CHERIBSD_TARGETS_WITH_HYBRID_FOR_PURECAP_ROOTFS + [
        CompilationTargets.NATIVE]`

Reply:
We do actually run hybrid tests on a hybrid rootfs.

Reply:
Is there a specific reason you are using a hybrid rootfs or could you change over to a purecap one?

Reply:
The original reason was simply that it seemed like the obvious, simplest thing to do. However, it does seem that we can run our hybrid builds perfectly well with `morello-webkit-morello-hybrid-for-purecap-rootfs`.

Given that, and also that @jrtc27 wanted to rewrite it differently anyway, I propose just closing this PR.